lib/pana.dart

```diff
@@ -142,7 +142,14 @@ class PackageAnalyzer {
 
     if (summary != null) {
       try {
-        libraryScanner = new LibraryScanner(pkgDir, isFlutter);
+        var overrides = {
```

Comment:
We can already detect that in a generic way, not specific to any given package. Do we need something over that?